driver/bpf/fillers.h

```diff
@@ -4678,6 +4711,16 @@ FILLER(tcp_drop_e, false)
 
 }
 
+FILLER(tcp_connect_x, false)
+{
+
```

Comment:
please add line like `return 0` in case that compiler complains, together with other empty FILLER.